.github/zizmor.yml

```diff
@@ -10,3 +10,8 @@ rules:
     ignore:
       - build-docker.yml
       - publish-playground.yml
+  excessive-permissions:
+    ignore:
```

Comment:
It might be helpful to add a short comment explaining why they ignored. Ideally, this wouldn't be necessary (maybe add a todo?)

Reply:
there was already a TODO on line 4, but I added a comment closer to these lines explaining exactly why they are ignored :-)